plugins/permission-backend/src/service/router.ts

```diff
@@ -188,7 +188,13 @@ export async function createRouter(
       req: Request<EvaluatePermissionRequestBatch>,
       res: Response<EvaluatePermissionResponseBatch>,
     ) => {
-      const user = await identity.getIdentity({ request: req });
+      let user;
+      try {
+        user = await identity.getIdentity({ request: req });
+      } catch (e) {
+        res.status(401).send();
```

Comment:
```suggestion
        res.status(401).end();
```

Reply:
Shouldn't we also propagate the message from the identityClient here?

Reply:
Thanks! I've amended it to do this:

`res.status(401).send({error: e.message});`

Reply:
Always use .json instead of .send to send json data (for security reasons).

Doesn't this router use the errorHandler middleware so that we can just throw a NotAllowedException here and let it deal with it?

However... Since this is a security feature, it may be the case that we explicitly do NOT want to propagate details about the error to the caller, since it makes attacks easier to formulate. So it could be the case that we should just go back to end() anyway - at least in production mode

Reply:
In that case, I think just HTTP 401 is sufficient.